appveyor.yml

```diff
@@ -0,0 +1,15 @@
+environment:
+  nodejs_version: "6"
+  GH_TOKEN:
+    secure: fyB6CRcrHbroxaBvzN6aPHUEhHGc3ljbbTPtEqruaDVe/iO8/dZn4LOVNS/fAlSX
```

Comment:
is it safe to disclose the token for the public?

Reply:
It is yeah. It's an encrypted token. Only AppVeyor can decrypt it.